### ISSUE TITLE
Add DOIs to publications

### ISSUE THREAD

**Issue by lunebellec** [neurolibre/neurolibre.com#11]
We should create and link DOIs for each publication / version 

https://blog.scholasticahq.com/post/getting-started-with-dois-at-your-journal-interview-with-anna-tolwinska-crossref/

**Comment by agahkarakuzu** [comment]
@pbellec do you feel strongly about issuing DOI with crossref? I was thinking about Zenodo.

**Comment by emdupre** [comment]
If we're counting publication as a version-cut, then Zenodo has an existing GitHub integration already to issue a DOI for each version. That's probably what you're thinking of, @agahkarakuzu ?

Distill doesn't cut versions, and they use Crossref. I think I still have an email from Chris Olah on their internal process, if we wanted to use that ! But if we're cutting versions then I don't know that it's as necessary.

**Comment by agahkarakuzu** [comment]
> That's probably what you're thinking of, @agahkarakuzu ?

Yes, this is exactly what I had in mind. First, it is practical. Second, I find it reasonable to have cut-versions, given that the citation refers to an immediate output of code execution, more easily subjected to change compared to a traditional publication with a fixed DOI.

On the other hand, I can also see the benefits of having a fixed DOI. Especially when I think about an altmetric badge.

**Comment by ltetrel** [comment]
Interresting ref : https://github.com/jupyter/jupyter/issues/190
